src/collection-browser.ts

```diff
@@ -41,13 +41,16 @@ import {
   defaultSelectedFacets,
   TileModel,
   CollectionDisplayMode,
+  EmptyPlaceholderType,
 } from './models';
 import {
   RestorationStateHandlerInterface,
   RestorationStateHandler,
   RestorationState,
 } from './restoration-state-handler';
 import chevronIcon from './assets/img/icons/chevron';
+import noSearchTerm from './assets/img/icons/no-search-term';
```

Comment:
Suggestion: You are using `import` aliasing here. Please rename these images to something like `imageNoSearchTerm` or `noSearchTermIcon`, `noSearchTermImage` to allow readability of intent in template code below.